helios-services/src/main/java/com/spotify/helios/agent/AgentConfig.java

```diff
@@ -369,4 +371,13 @@ public AgentConfig setConnectionPoolSize(final int connectionPoolSize) {
     this.connectionPoolSize = connectionPoolSize;
     return this;
   }
+
+  public File getGoogleCloudAccountCredentials() {
```

Comment:
nitpick: I think it would make more sense to have this be of type `GoogleCredentials` and load the credential file elsewhere (e.g. in `AgentParser`).

That would make it easier to add a flag like `--docker-gcp-use-application-default-credentials` in the future if we want to do that.